pkgs/tools/admin/lxd/ui.nix

```
@@ -0,0 +1,42 @@
+{ mkYarnPackage
+, fetchFromGitHub
+, fetchYarnDeps
+, lib
+, ...
+}:
+mkYarnPackage rec {
+  pname = "lxd-ui";
+  version = "0.0.1";
```

```suggestion
  version = "unstable-2023-07-03";
```

Upstream does not yet have a version for this (if they ever will) so we need to version this as unstable per the manual.